### PR TITLE
Auth view aesthetic updates

### PR DESCRIPTION
**Pull request by ackernaut**
Addresses #1848

### CODE REVIEW

File: src/universal/components/FieldLabel/FieldLabel.js [hunk 1]
```diff
@@ -40,12 +39,14 @@ const styleThunk = (theme, {customStyles, fieldSize, indent, inline}) => {
...
-      color: appTheme.palette.dark,
+      color: ui.labelHeadingColor,
       display: 'block',
-      fontSize: appTheme.typography.sBase,
-      fontWeight: 600,
-      lineHeight: appTheme.typography.s5,
+      fontSize: ui.labelHeadingFontSize,
+      fontWeight: ui.labelHeadingFontWeight,
+      lineHeight: ui.labelHeadingLineHeight,
+      letterSpacing: ui.labelHeadingLetterSpacing,
```
Comment (dan-f) [inline src/universal/components/FieldLabel/FieldLabel.js]: we need an "ah, cool, i am learning" emoji

File: src/universal/components/SignInPage/SignInEmailPasswordForm.js [hunk 2]
```diff
@@ -28,9 +28,7 @@ const Form = styled('form')({
   flexDirection: 'column'
 });
 
-const FieldsContainer = styled('div')({
-  marginBottom: '2rem'
-});
+const Block = styled('div')(({margin, width}) => ({margin, width}));
```
Comment (dan-f) [inline src/universal/components/SignInPage/SignInEmailPasswordForm.js]: I like this utility you added! If you think it'll be a useful pattern going forward, let's add it to `universal/components/`

**Comment by dan-f** [comment]
Looks really great, presentationally and in code! My only nitpick is that, when posing as an iPhone in chrome, I'm able to zoom out past the dimensions of the root node. Not sure if this is a bug on our end or in chrome's devtools. I'm happy to kick this can down the road and test it on a real phone once it's on a dev server/staging, and prioritize separately unless it's got a straightforward or simple solution.

screen shot 2018-03-21 at 6 41 50 pm

File: src/universal/components/SignInPage/SignInEmailPasswordForm.js (hunk 2 above)
Comment (ackernaut) [reply src/universal/components/SignInPage/SignInEmailPasswordForm.js]: Good point…I may just do that.

**Comment by ackernaut** [comment]
@dan-f I’ll take a look at the layout at those breakpoints. Should be a simple fix. So he said…

File: src/universal/components/AuthPage/AuthPage.js [hunk 3]
```diff
@@ -33,6 +31,8 @@ const CenteredBlock = styled('div')({
   flexDirection: 'column',
   alignItems: 'center',
   height: '100%',
+  maxWidth: '100%',
```
Comment (ackernaut) [inline src/universal/components/AuthPage/AuthPage.js]: @dan-f this should take care of our overflow woes down at mobile device widths for now.

**Comment by ackernaut** [comment]
Hrm 🤔 now `e2e` tests seem to be failing…do we need dependency updates @dan-f ?

**Comment by ackernaut** [comment]
@dan-f thanks for the stellar effort and review on auth views! was very clean, easy to follow along and give 1 more aesthetic pass…

**Comment by dan-f** [comment]
Glad to hear it! Thank you for making it even better!

**Pull request merged**